### ISSUE TITLE
[Request] sigma-file-manager

### ISSUE THREAD

**Issue by Pakrohk** [chaotic-aur/packages#2377]
### Package

https://aur.archlinux.org/pkgbase/sigma-file-manager

### Purpose

A free, open-source, quickly evolving, modern file manager (explorer / finder) app for Windows and Linux.

### License

GPL-3.0

**Comment by Technetium1** [comment]
This is a rather large appimage @ 200MB. I don't know if we should add something so large.

**Comment by Pakrohk** [comment]
> Mhmm, I would agree that this being an Appimage + 200MB is rather tedious.

I agree that the size of the current package that comes from appimage is really high.
I will open an issue in the GitHub of the main project and request that they create a more suitable package in aur, something that specifies the dependencies instead of appimage.

**Comment by begin-theadventure** [comment]
I made a bin package, the size of .tar.zst is only 8 MB smaller.
This application uses Electron, so that's why the size is large.

**Comment by dr460nf1r3** [comment]
> And honestly I wonder why chaotic even hosts Electron bin apps and bin apps in general, a bit pointless since most bin packages take little time to install with the AUR helpers. Example: visual-studio-code-bin 349,8 MB.

It's convenient to have those if it is a widespread application, your example would fit this case.

**Comment by xiota** [comment]
Closing because upstream still uses Electron 20.1.3.  Last commit was 10 months ago.  Last release was Jun 2023.
